gradle/dependency-versions.gradle

```diff
@@ -30,7 +30,7 @@
   metricsVersion = "2.2.0"
   kafkaVersion = "0.10.0.1"
   commonsHttpClientVersion = "3.1"
-  rocksdbVersion = "3.13.1"
```

Comment:
SAMZA-964 (continuous checkpointing for stores) seems to be unrelated to this upgrade. Is it a typo?